test/unit/test-http-handler.js

```diff
@@ -0,0 +1,209 @@
+var assert = require('chai').assert;
+var sinon = require('sinon');
+var crypto = require('crypto');
+var proxyquire = require('proxyquire');
+var correctRawBody = 'payload=%7B%22type%22%3A%22interactive_message%22%7D';
+
+var getRawBodyStub = sinon.stub();
+var systemUnderTest = proxyquire('../../dist/http-handler', {
+  'raw-body': getRawBodyStub
+});
+var createHTTPHandler = systemUnderTest.createHTTPHandler;
+// fixtures
+var correctSigningSecret = 'SIGNING_SECRET';
+var requestSigningVersion = 'v0';
+
+function createRequestSignature(signingSecret, ts, rawBody) {
```

Comment:
the next 2 functions should prob go in `test/helpers.js` (and if necessary, shared with `test/unit-test-adapter.js`)